app/elements/socobo-element-toast/socobo-element-toast.html

```diff
@@ -0,0 +1,142 @@
+<!--
+The following custom properties and mixins are available for styling:
+
+Custom property | Description | Default
+----------------|-------------|----------
+`--socobo-toast-style-error-notification-color` | Set background color for socobo-toast error notification | `{#EEEEEE}`
+`--socobo-toast-style-success-notification-color` | Set background color for socobo-toast success notification | `{#EEEEEE}`
+`--socobo-toast-style-plain-notification-color` | Set background color for socobo-toast info notification | `{#EEEEEE}`
+`--socobo-toast-style-spinner-notification-color` | Set background color for socobo-toast spinner notification | `{#EEEEEE}`
+`--socobo-recipe-style-notification-text-color` | Set the text color for the toasts | `{}`
+
+@demo demo/socobo-element-toast/demo.html
+-->
+<dom-module id="socobo-element-toast">
+  <template>
+    <style>
+      :host {
+        display: block;
+        font-family: "Roboto", "Helvetica Neue", Helvetica, Arial, sans-serif;
+      }
+
+      .socobo-toast-error {
+        --paper-toast-background-color: var(--socobo-toast-style-error-notification-color);
+        --paper-toast-color: var(--socobo-recipe-style-notification-text-color);
+      }
+      .socobo-toast-success {
+        --paper-toast-background-color: var(--socobo-toast-style-success-notification-color);
+        --paper-toast-color: var(--socobo-recipe-style-notification-text-color);
+      }
+      .socobo-toast-plain {
+         --paper-toast-background-color: var(--socobo-toast-style-plain-notification-color);
+         --paper-toast-color: var(--socobo-recipe-style-notification-text-color);
+       }
+      .socobo-toast-spinner {
+          --paper-toast-background-color: var(--socobo-toast-style-spinner-notification-color);
+          --paper-toast-color: var(--socobo-recipe-style-notification-text-color);
+        }
+    </style>
+
+    <div>
+      <paper-toast class="socobo-toast-plain" id="toast"></paper-toast>
+    </div>
+  </template>
+
+  <script>
+    (function() {
+      Polymer({
+        is: "socobo-element-toast",
+
+        properties: {
+          _messages: {
+            type: Array,
+            value: []
+          }
+        },
+
+        observers: [
+          "_messageAdded(_messages.splices)"
+        ],
+
+        _DEFAULT: 3000,
+        /**
+         * Handles the global toast events and decides which toast to show when
+         *
+         * @param {CustomEvent} e
+         * @private
+         * @function
+         */
+        _showToast: function(e) {
+          this.push("_messages", e.detail);
+        },
+
+        /**
+         * Add the correct style to the toast and remove any previously set ones
+         *
+         * @param {DomElement} toast
+         * @param {Object} msg
+         * @private
+         */
+        _styleToast: function(toast, msg) {
+          Array.from(toast.classList).filter(function(clazz) {
```

Comment:
No, but it has been implemented in Chrome since version 45 and FF since 35. Also works in Edge. So I do not see any problems with that.

It would be possible to add a reduced Polyfill, aimed at this specific use case:

```javascript
function polyfill(arrayLike) {
  var arr = [];
  arrayLike.forEach(function(e) {
    arr.push(e);
  }):
  return arr;
});
```

Of course, that would only work in this specific case. But that is the point, right. It would make more sense to rewrite this part to something not using the `DomTokenList` returned by `Element.classList` at all.

According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList), `Element.className` returns a string with all classes separated by spaces, so this would work:

```javascript
  toast.className.split(" ").filter(...)
```

As long as there is no actual problem with the implementation of `Array.from``` on any Browser that we target, I don't see any reason to change that.